### DISCUSSION TITLE
Labs coming to the new Hero?

### DISCUSSION THREAD

**Discussion by legitleavitt** [gopro/labs#1125] [Q&A]
Considering buying the new Hero (the itty bitty guy, annoying when they don't use numbers/names w/ releases because googling questions is basically impossible), but being able to do timed-start is necessary for what I do.

Anyone know if either 
A) Labs is coming to the new Hero,
B) Timed/delayed start is built in?

**Comment by dnewman-gpsw** [comment]
A) unlikely any time soon, maybe in the distant future.  HERO 4K is intentionally very simple, will little to no custom settings, Labs is currently to opposite of that. 
B) not built it.

**Reply by legitleavitt** [reply]
Gotcha, a bummer, but totally makes sense. Thanks for clarifying that for me.
